src/lib/mina_lib/mina_lib.ml

```diff
@@ -2385,20 +2395,30 @@ let create ~commit_id ?wallets (config : Config.t) =
                       loop ()
                   | Ok `Synced
                     when equal_status !last_sync_status `Catchup
-                         || equal_status !last_sync_status `Bootstrap ->
-                      [%log' info config.logger]
-                        ">>>>>> Synced, fetching completed snarks" ;
-                      let top_block =
-                        let open Mina_block in
```

Comment:
The logic here should remain the same, right?  Otherwise you are checking that the node is progressing but not necessarily nearing the latest block in the network?

Reply:
you are right, i thought this would be fine since we would be validating against data from the same pipe and we fetch best tip from the seed node, but that is not necessarily guaranteed to be in the transition frontier.

Reply:
done